connectors/es/client.py

```diff
@@ -47,9 +46,7 @@ class ESClient:
     user_agent = f"{USER_AGENT_BASE}/service"
 
     def __init__(self, config):
-        # We don't have a way to ask the server, but it's planned
-        # for now we just use an env flag
-        self.serverless = "SERVERLESS" in os.environ
```

Comment:
Do we have docs anywhere for this SERVERLESS env var that we need to edit/remove?

Reply:
It's not referenced in our docs for ingesting into serverless: https://www.elastic.co/docs/current/serverless/elasticsearch/ingest-data-through-integrations-connector-client
Kind of surprised at that!

Reply:
I bet it was originally for the serverless CI pipeline we had.